services/src/main/java/edu/unc/lib/dl/cdr/services/rest/modify/MoveObjectsController.java

```diff
@@ -49,23 +45,10 @@
 public class MoveObjectsController {
     private static final Logger log = LoggerFactory.getLogger(MoveObjectsController.class);
 
-//    @Autowired
-//    private TripleStoreQueryService tripleStoreQueryService;
-//    @Autowired
-//    private DigitalObjectManager digitalObjectManager;
-
-    private final Long TIME_TO_LIVE_AFTER_FINISH = 12000L;
-
-    private final Map<String, MoveRequest> moveRequests;
-
-    @Autowired
-    private ExecutorService moveExecutor;
-
     @Autowired
-    private ActivityMetricsClient operationMetrics;
+    private MoveObjectsService moveObjectsService;
 
     public MoveObjectsController() {
-        moveRequests = new HashMap<>();
     }
 
     @RequestMapping(value = "edit/move", method = RequestMethod.POST)
```

Comment:
i think it might be a good idea to have this controller's organization match what we've been doing with our other controllers recently. so, e.g., moveObjects() should call a private method within the controller that would itself return a ResponseEntity. also, at line 60 there is an opening '{' that never gets closed. and while we're at it, maybe remove the unnecessary default constructor for MoveRequest at line 94

Reply:
I usually try to split out to a private method if the content of the method is going to be shared by more than one controller endpoint.  Since there's only one controller method currently and it is just doing controllery stuff, I'm inclined to leave it, but I'd be happy to refactor this later if we have multiple controller methods in this class that need the same functionality though.

Went ahead and removed that default constructor.

Reply:
ok, that's fine. how would you feel about at least changing the return type to be a ResponseEntity, and to use predefined constants for the response codes? then you could also remove HttpResponse as a param to the controller method. i just think that this ```return new ResponseEntity<>(result, HttpStatus.OK)``` looks much cleaner than this '''response.setStatus(200);``` followed by ```return results;```

Reply:
Ah good call, i mentally skipped over that big in the first comment, sorry.  Should be addressed now